packages/external-db-config/lib/service/config_reader.js

```diff
@@ -1,31 +1,44 @@
 class ConfigReader {
-  constructor(externalConfigReader, commonConfigReader) {
+  constructor(externalConfigReader, commonConfigReader, externalAuthConfigReader) {
     this.externalConfigReader = externalConfigReader
+    this.externalAuthConfigReader = externalAuthConfigReader
     this.commonConfigReader = commonConfigReader
   }
 
   async readConfig() {
-    return await this.externalConfigReader.readConfig()
+    const externalConfig = await this.externalConfigReader.readConfig()
+    const authConfig = await this.externalAuthConfigReader.readConfig()
+
+    return { ...externalConfig, auth: authConfig }
   }
 
   async configStatus() {
     const { missingRequiredSecretsKeys } = await this.externalConfigReader.validate()
-    const { missingRequiredSecretsKeys: missing, validType, validVendor } = this.commonConfigReader.validate()
+    const { missingRequiredSecretsKeys: missingRequiredAuthSecretsKeys } = await this.externalAuthConfigReader.validate()
+    const { missingRequiredSecretsKeys: missingRequiredEnvs, validType, validVendor } = this.commonConfigReader.validate()
 
-    if (missingRequiredSecretsKeys.length > 0 || (missing && missing.length > 0)) {
-      return `Missing props: ${[...missingRequiredSecretsKeys, missing].join(', ')}`
-    }
+    const validConfig = missingRequiredSecretsKeys.length === 0 && missingRequiredEnvs.length === 0 && validType && validVendor
+    const validAuthConfig = missingRequiredAuthSecretsKeys.length === 0
     
-    else if (!validVendor) {
-      return 'Cloud type is not supported'
-    }
-
-    else if(!validType) {
-      return 'DB type is not supported'
-    }
+  
+    let message 
```

Comment:
The login screen and app info will use `configStatus` to print the message if something is missing,
Hence, the order of the ifs ....